AllReadyApp/Web-App/AllReady/DataAccess/AllReadyDataAccessEF7.ActivitySignup.cs

```diff
@@ -39,24 +39,31 @@ Task IAllReadyDataAccess.AddActivitySignupAsync(ActivitySignup userSignup)
             return _dbContext.SaveChangesAsync();
         }
 
-        Task IAllReadyDataAccess.DeleteActivitySignupAsync(int activitySignupId)
+        Task IAllReadyDataAccess.DeleteActivityAndTaskSignupsAsync(int activitySignupId)
         {
             var activity = _dbContext.ActivitySignup.SingleOrDefault(c => c.Id == activitySignupId);
 
-            if (activity != null)
+            if (activity == null) return null;
```

Comment:
Although technically correct, the way the code is formatted makes your intentions somewhat unclear ("do you mean to return null when activity is null or execute the block of code?"). Something like this would be better:

``` CSharp
if (activity == null)
{
  return null; //Purposely on its own line to draw attention to the fact that this is an exit point
}

//The real work...
```